packages/connectivity/connectivity_macos/macos/Classes/ConnectivityPlugin.swift

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Google LLC
```

Comment:
Should we be writing these in Swift or Objective C? I don't mind Swift, but so far we've been avoiding it to avoiding adding an N+1 language burden to the repo. /cc @amirh 

If we're writing the macOS plugins in Swift I feel like that's more of a reason to also write the mobile ones in Swift, too.

Reply:
I believe the preferred language for macOS plugins is Swift. /cc @stuartmorgan 

I don't think it would be worth rewriting the iOS ones since they interoperate and would be a lot of work with small benefit.

Reply:
Swift is the preferred language for macOS plugins; there isn't even an ObjC template in the tool. It's possible to write ObjC plugins, but we should be doing what we are recommending to users.

Whether new iOS plugins, if any, should use Swift is a separate decision. On macOS there's no legacy argument, so the decision is much clearer.